ProblemsSolved.js

```diff
@@ -0,0 +1,76 @@
+function ProblemsSolved() 
+{
+}
+
+ProblemsSolved.prototype.SumOfEvenNumberInArray = function(array)
+{
+    var sum = 0;
+    for(i=0; i < array.length; i++)
+    {
+		if(array[i]%2 == 0)
+			sum += array[i];
+    }
+    return sum;
+}
+
+ProblemsSolved.prototype.GetTheLargestTwoNumber = function(array)
+{
+    var pairNumbers = [];
+    pairNumbers[0] = -Infinity;
+    pairNumbers[1] = -Infinity;
+	
+	var localMaximum = -Infinity;
+	var indexOfMaximumNumber = 0;
+	
+	if(array.length < 2)
+		return 'Array length is to small. Give an array with minimum 2 pairNumbers';
+	
+	if(array.length == 2)
+	{
+		if(array[0] > array[1])
+		{
+			var temp = array[1];
+			array[1] = array[0];
+			array[0] = temp;
```

Comment:
No requirement specified in this problem. The only restriction was: "Do not sort the array.". I determined that the previous maximum be the first  and the current maximum to to be second, in that case. This helped me at unit testing because I knew which element is the maximum and current.